sdk/eventhub/Azure.Messaging.EventHubs/src/Authorization/EventHubSharedKeyCredential.cs

```diff
@@ -72,6 +78,28 @@ public sealed class EventHubSharedKeyCredential : TokenCredential
         ///
         public override ValueTask<AccessToken> GetTokenAsync(TokenRequestContext requestContext, CancellationToken cancellationToken) => throw new InvalidOperationException(Resources.SharedKeyCredentialCannotGenerateTokens);
 
+        /// <summary>
+        ///   Allows the rotation of Shared Access Signatures.
+        /// </summary>
+        ///
+        /// <param name="keyName">The Event Hubs resource to which the token is intended to serve as authorization.</param>
+        /// <param name="keyValue">The duration that the signature should be considered valid; if not specified, a default will be assumed.</param>
+        /// <param name="signatureValidityDuration">The duration that the signature should be considered valid; if not specified, a default will be assumed.</param>
+        ///
+        public void UpdateSharedAccessKey(string keyName, string keyValue, TimeSpan? signatureValidityDuration = default)
```

Comment:
nit:  If you would please be so kind, split parameters one to a line for consistency with surrounding code.